src/styles/_catalog.scss

```
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, $card-size);
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, $card-size);
+    }
+  }
```

it can be simplified
```suggestion
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, $card-size);
  }
```